### DISCUSSION TITLE
snmp-exporter does not collect Memory and CPU metrics from fortigate

### DISCUSSION THREAD

**Discussion by moisesaugustoa** [prometheus/snmp_exporter#1286] [Q&A]
I implemented the chart using helm, within the snmp.yml file there are several modules, I am using if_mibs, I tried using others, but it does not provide memory and cpu metrics. I tried to upload a configmap and pass it to extraConfigmapMounts []: it still doesn't bring memory metrics. I would really like help on how to bring these metrics

**Comment by bastischubert** [comment]
hi @moisesaugustoa ,
i guess you need to build your own snmp.yml - the discussion https://github.com/prometheus/snmp_exporter/discussions/1151 should be helpful for your use-case

**Reply by moisesaugustoa** [reply]
Hi @bastischubert,
I created a module with the mibs I need, but it still isn't collecting them.

**Reply by bastischubert** [reply]
do you mind sharing 

- your generator.yml
- your resulting snmp.yml 
- the output of an snmpwalk of the fortigate where we can see the the values returned for 1.3.6.1.4.1.12356.101.4.1.3
- the output of snmp_exporter when run with debug log level
